compiler/rustc_codegen_llvm/src/back/write.rs

```diff
@@ -98,7 +99,7 @@ pub fn write_output_file<'ll>(
     }
 }
 
-pub fn create_informational_target_machine(sess: &Session) -> &'static mut llvm::TargetMachine {
```

Comment:
uh, I wasn't aware that this code uses `'static` here and then drops it anyways, that is indeed pretty bad. (lying about lifetimes isn't undefined behavior but its a great step towards it by making the code confusing).